Sources/MapboxNavigation/InstructionLabel.swift

```diff
@@ -33,6 +33,13 @@ open class InstructionLabel: StylableLabel, InstructionPresenterDataSource {
         }
     }
 
+    open override func update() {
+        super.update()
```

Comment:
There is still one case of https://github.com/mapbox/mapbox-navigation-ios/issues/2938 where this doesn't work optimally. If you are currently viewing a UIView that that contains an InstructionLabel, such as the Step List, the cache isn't invalidated (and thus the colors are not updating) if you toggle light/dark mode. If you dismiss the UI and bring it back up then the colors will be correct.

You can add support for the `traitCollection` changing and trigger an invalidation there as well. For example, you could add: 

`open override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
        super.traitCollectionDidChange(previousTraitCollection)
        update()
    }`

to `InstructionLabel.swift` as well.

Reply:
Thanks for the suggestion! I'll work on this case.